docs/user-guide/kubernetes.md

```diff
@@ -332,24 +332,24 @@ spec:
   selector:
     k8s-app: traefik-ingress-lb
   ports:
-  - port: 80
+  - name: web
+    port: 80
     targetPort: 8080
 ---
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: traefik-web-ui
   namespace: kube-system
-  annotations:
-    kubernetes.io/ingress.class: traefik
 spec:
```

Comment:
The annotation makes sense to have in general. I'm thinking that we should rather add it to the YAML file than remove it from the inline spec.

@dtomcej WDYT?

Reply:
The argument is that we don't specify an ingress class to watch on any of our examples. If we keep it, we _should_ be adding the class to our deployment/daemonset...

Reply:
Gotcha. Even if we decided to do that, it should be done by a separate PR. No reason to block this one.